parser-typechecker/src/Unison/Codebase/Editor/Input.hs

```diff
@@ -103,7 +103,7 @@ data Input
   | LinksI Path.HQSplit' (Maybe String)
   -- other
   | SearchByNameI Bool Bool [String] -- SearchByName isVerbose showAll query
-  | FindShallow Path'
+  | FindShallowl Path'
```

Comment:
I think this should be `FindShallowI`, i.e. the last letter is `I` (uppercase `i`), not lowercase `L`.

Reply:
@TomasMikula Thanks for catching this.